libs/wire-api/src/Wire/API/Conversation.hs

```diff
@@ -153,11 +153,10 @@ conversationMetadataObjectSchema =
       .= optional (field "last_event_time" schema)
     <*> cnvmTeam .= optField "team" (maybeWithDefault A.Null schema)
     <*> cnvmMessageTimer
-      .= ( optFieldWithDocModifier
-             "message_timer"
-             (description ?~ "Per-conversation message timer (can be null)")
-             (maybeWithDefault A.Null schema)
-         )
+      .= optFieldWithDocModifier
+        "message_timer"
+        (description ?~ "Per-conversation message timer (can be null)")
+        (maybeWithDefault A.Null schema)
```

Comment:
@battermann and this is why you should not mix hlint changes and feature work in the same PR!  :)  (see #2035)